application/lib/view/widgets/AppBar.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:application/routes.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class CustomAppBar extends StatelessWidget implements PreferredSizeWidget {
+  CustomAppBar({this.centerTitle = false, this.title = "No title"});
```

Comment:
Kan ikke se hvorfor at der skal stå no title, hvis der ikke gives en title med. Tænker at hvis der ikke gives det med, så er det fordi der ikke skal stå noget.
```suggestion
  CustomAppBar({this.centerTitle = false, this.title = ""});
```